app/content/plan/index.js

```diff
@@ -1,14 +1,205 @@
+                <View
+                  style={{
+                    borderWidth: 1,
+                    borderColor: 'rgba(0,0,0,0.2)',
+                    alignItems: 'center',
+                    justifyContent: 'space-between',
+                    width: '100%',
+                    height: 50,
+                    backgroundColor: `${!drill.completed ? '#fff' : 'green'}`,
+                    borderRadius: 50,
+                    marginBottom: 10,
+                    paddingLeft: 30,
+                    paddingRight: 30,
+                    flexDirection: 'row',
+                  }}
+                >
+                  <Text style={{ fontSize: 20 }}>{drillInfo[drill.drill]["drillType"]}</Text>
+                  <Text style={{ fontSize: 20 }}>{drillInfo[drill.drill]["spec"]}</Text>
+                </View>
+              </View>
+            </TouchableOpacity>
+          ))}
+        </View>
+      )}
+    />
+  );
+};
+
+const convertTimestampToDate = (timestamp) => {
+  const date = new Date(timestamp * 1000);
+  const year = date.getFullYear();
+  const month = String(date.getMonth() + 1).padStart(2, '0');
+  const day = String(date.getDate()).padStart(2, '0');
+  return `${year}-${month}-${day}`;
+};
+
+
+
+
+
 
-//This is for the list of drills
 export default function Index() {
+  const d = doc(db, "teams", "1", "drills", "SpvYyY94HaulVH2zmVyM")
+  //getDoc(d).then(value => console.log(value.data()));
+
+  const [cardDataTest, setCardDataTest] = React.useState([])
+  useEffect(() => {
+    let drillList = [];
+    let promiseList = [];
+    /*Object.values(assigned_data).forEach((data) => {
+      data.drill = getDoc(doc(db, "teams", "1", "drills", data.drill)).then(drillData => (
+        drillData.data()
+      ))
+    });
+    console.log("DRILL LIST", drillList)
+    Object.values(assigned_data).forEach((data) => {
+      console.log("DATA", data)
+      drillList.push(data.drill);
+    });*/
+
+    /* assigned_data.drills.forEach(did => {
+       promiseList.push(
+         getDoc(doc(db, "teams", "1", "drills", did)).then(drillData => (
+           drillData.data()
+         ))
+       );
+ 
+     });
+ 
+     Promise.all(promiseList).then(values => {
+       values.forEach((data) => {
+         console.log("DRILL DATA", data);
+         drillList.push(data);
+       });
+       setCardDataTest(drillList);
+     });*/
+  }, []);
+  console.log("TEST CARD DATA", cardDataTest)
+  console.log(formatDate(1710292140))
+
+
   return (
     <PaperProvider>
       <SafeAreaView
-        style={{ flex: 1, justifyContent: "center", alignItems: "center" }}
+        style={{ flex: 1 }}
+        edges={["right", "top", "left"]}
       >
-        <Text>Plan Index</Text>
+        <Appbar.Header statusBarHeight={0} style={{ backgroundColor: "FFF" }}>
+
+          <Appbar.Content title={"Plan"} titleStyle={{ fontWeight: 'bold' }} />
+        </Appbar.Header>
+
+        <DrillList></DrillList>
```

Comment:
Just merge DrillList with Input, there's no point having these 2. Or even better, put "PaperProvider", and "SafeAreaView" inside Drill List and remove Index